src/Apps/Sell/Routes/ThankYouRoute.tsx

```diff
@@ -44,14 +44,14 @@ export const ThankYouRoute: React.FC<ThankYouRouteProps> = props => {
             <Join separator={<Spacer y={2} />}>
               <SubmissionStepTitle>
                 {isSubmitted
-                  ? "Thank you for submitting your artwork"
-                  : "Thank you for listing your artwork"}
+                  ? "Thank you for listing your artwork"
```

Comment:
https://www.figma.com/design/E6rVm1tiSI4PW3EzibKKZw/Submission-%26-Pre-Consignment-Management?node-id=2036-30168&m=dev

```suggestion
                  ? "Thank you for submitting your artwork"
```